system/jlib/jstring.hpp

```diff
@@ -588,6 +588,7 @@ inline StringBuffer& operator << (StringBuffer& s, const TValue& value)
 }
 
 extern jlib_decl void toLower(std::string & value);
+extern jlib_decl void trim(std::string & value);
```

Comment:
could do with comment.. what does it do? Looks like trims leading white space only not trailing